go.mod

```diff
@@ -1,13 +1,48 @@
 module github.com/flip-id/wappin
 
-go 1.14
+go 1.17
```

Comment:
mas just want to ask, currently we using go 1.13 on notification service, it would be impacted ?

Reply:
@asqolani-flip No mas, it's safe, since `valuefirst` package also uses the go 1.17. Reference: https://github.com/flip-id/valuefirst/blob/main/go.mod#L3.